### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by PGijsbers**

### CODE REVIEW

**Comment by PGijsbers** [comment]
Removed the `TODO.md` file. Saved the content in Github issues #169, #170, #171, and #172.
The following points are not covered in issues (yet):
 - "progress bar?? fancy useless stuff.", perfectly fine to open an issue for it, though I would personally wait until someone requests it (or wants to work on it).
 - "properly kill job threads on KeyInterruptError (mainly in AWS mode)", I wasn't sure if this was added yet. If not, I think it makes sense to open an issue for this.
 - "more pydoc" and "unit tests", I feel opening a separate issue is not required, #165 already discusses that we want to work on the maturity of the codebase, which include these. #166 also sets up travis and a start to unit tests.

File: docs/README.md [hunk 1]
```diff
@@ -337,3 +339,24 @@ Any new issue should also be reported there.
...
 [boto3]: https://boto3.readthedocs.io/
+
+
+## Frequently Asked Questions
+
+**When will results be updated, also for the new/updated frameworks?**
+
+We don't perform a benchmark evaluation for each new package or update.
+Due to budget constraints, we can only do a limited number of evaluations.
+The next full evaluation will be performed before the end of the year.
```
Comment (sebhrusen) [inline docs/README.md:350]:
```suggestion
The next full evaluation will be performed before the end of the year 2020.
```

**Review by sebhrusen — APPROVED** [review]
thx

**Comment by sebhrusen** [comment]
Given https://github.com/openml/automlbenchmark/issues/165, we should probably also update the setup section:
```
### Setup
Clone the repo:
```bash
git clone https://github.com/openml/automlbenchmark.git --branch stable
cd automlbenchmark
```

**Comment by sebhrusen** [comment]
@PGijsbers 
> "properly kill job threads on KeyInterruptError (mainly in AWS mode)", I wasn't sure if this was added yet. If not, I think it makes sense to open an issue for this.

this was fixed by https://github.com/openml/automlbenchmark/pull/138

**Pull request merged**